docs/illink-tasks.md

```diff
@@ -1,5 +1,13 @@
 # ILLink.Tasks
 
+## Terminology
+
+The capability of the SDK is called "trim". Trimming allows the SDK to reduce the size of the application by removing unused code and other assets. MSBuild/SDK integration should use "trim" in all user facing names (property names, item group names, ...).
```

Comment:
```suggestion
The capability of the SDK is called "trim". Trimming allows the SDK to reduce the size of the application by removing unused code and other assets. MSBuild/SDK integration should use "trim" in all user facing names specific to trimming (property names, item group names, ...).
```

Nit: since we do have some integration points that technically are user-facing for advanced users (ex: `PrepareForILLink`) that relate to the tool, maybe be a little more specific.

Reply:
I agree the wording is a bit confusing. My suggestion

```suggestion
Trimming allows a developer to reduce the final size of the application by removing unused managed code (IL) and other assets. When working with MSBuild properties or items groups any publicly visible name which is related to trimming starts with `Trim` prefix.

The trimming process is managed by MSBuild based ILLink.Tasks assembly which calls into ILLink tool to perform the actual trimming. The ILLink tool handles all the heavy work to produce trimmed output and can issue its own [warnings and errors](error-codes.md). All of them use `ILLink` prefix and unique code for easier identification. 
```

Reply:
I ended up using a variation of what @marek-safar suggested - thanks!